### ISSUE TITLE
Linux version

### ISSUE THREAD

**Issue by gromain** [madebybright/te#18]
Hi!

I love your app, any plans or interests for a Linux version?

**Comment by ethel-dev** [comment]
Yes, actually! We're gonna have to set up a Linux environment but, its coming ASAP. Windows too.

**Comment by gromain** [comment]
Great! Can't wait! I can play the alpha/beta tester if you want! I'm on Manjaro (an Arch Linux derivative)!

**Comment by ctian1** [comment]
Looking forward to a Windows version as well, if testing is needed

**Comment by susw12** [comment]
Has there been any progress on a Linux version? I am really looking forward to this.

**Comment by ethel-dev** [comment]
@susw12 Unfortunately, there hasn't been any hard progress. We are currently in the planning stages of a full rewrite of Nimble which we'd like to include multiplatform support in. Thanks for being patient! haha

**Comment by ethel-dev** [comment]
@susw12 I recently switched to openSUSE. I feel your pain. Work on a Linux port is underway!

**Comment by gromain** [comment]
Any update on this? It's been a while, is the project abandoned?
